### PR TITLE
pydantic-core 2.0.1

### PR DESCRIPTION
**Pull request by pavelzw**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+  sha256: f9fffcb5507bff84a1312d1616406cad157806f105d78bd184d1e6b3b00e6417
 
 build:
   script:
     # PyPy has weird sysconfigdata name
     - rm -f $PREFIX/lib/pypy$PY_VER/_sysconfigdata.py  # [build_platform != target_platform and target_platform == "linux-ppc64le"]
     - {{ PYTHON }} -m pip install . -vv
   number: 0
+  skip:
+    #
```
Comment (xhochy) [inline recipe/meta.yaml:19]: Please make an upstream issue about this.

**Review by xhochy — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (pavelzw) [reply recipe/meta.yaml:19]: https://github.com/pydantic/pydantic-core/issues/724

**Comment by 0xbe7a** [comment]
@conda-forge-admin, please rerender

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

**Pull request merged**